flexeval/core/language_model/openai_api.py

```diff
@@ -22,7 +45,9 @@ async def _retry_on_error(
             return await openai_call()
         except openai.APIError as e:  # noqa: PERF203
```

Comment:
how about moving `return EMPTY_RESPONSE` outside of the for-loop to reduce if-clauses?

Reply:
Please update the typing `-> Awaitable[T] | None` to `-> Awaitable[T]` as well : )